tests/regression/02-base/44-malloc_array.c

```diff
@@ -0,0 +1,11 @@
+// PARAM: --set ana.int.interval true --enable exp.partition-arrays.enabled
+#include<stdlib.h>
+#include<assert.h>
+
+int main(void) {
+    int *r = malloc(5 * sizeof(int));
+
+    r[3] = 2;
+
+    assert(r[4] == 2);
```

Comment:
This should have an explanation why this succeeds, it does not necessarily do so in the concrete.